modules/text2colbert-jinaai/module.go

```diff
@@ -0,0 +1,167 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package modjinaai
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/weaviate/weaviate/usecases/modulecomponents/batch"
+
+	"github.com/weaviate/weaviate/modules/text2vec-jinaai/ent"
+
+	"github.com/weaviate/weaviate/usecases/modulecomponents/text2vecbase"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/weaviate/weaviate/entities/models"
+	"github.com/weaviate/weaviate/entities/modulecapabilities"
+	"github.com/weaviate/weaviate/entities/moduletools"
+	"github.com/weaviate/weaviate/modules/text2vec-jinaai/clients"
+	"github.com/weaviate/weaviate/usecases/modulecomponents/additional"
+)
+
+const Name = "text2colbert-jinaai"
+
+var batchSettings = batch.Settings{
+	// the encoding is different than OpenAI, but the code is not available in Go and too complicated to port.
+	// using 30% more than the OpenAI model is a rough estimate but seems to work
+	TokenMultiplier:    1.3,
+	MaxTimePerBatch:    float64(10),
+	MaxObjectsPerBatch: 512, // Info from jina
+	// real limit is 8192, but the vectorization times go up by A LOT if the batches are larger
+	MaxTokensPerBatch: func(cfg moduletools.ClassConfig) int { return 2500 },
```

Comment:
I would double check that this is also true for colbert modules (all of the above)

Reply:
I think that those settings apply to the `/v1/embeddings` endpoint as a whole, so that should work, but I have made a note that we should double check this once this module will be actually working